expression/builtin_math.go

```diff
@@ -708,7 +708,25 @@ type builtinAcosSig struct {
 
 // See https://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_acos
 func (b *builtinAcosSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("acos")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return types.Datum{}, errors.Trace(err)
+	}
+	if args[0].IsNull() {
+		return args[0], nil
+	}
+	sc := b.ctx.GetSessionVars().StmtCtx
+	f, err := args[0].ToFloat64(sc)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	if f < -1 || f > 1 {
+		d.SetNull()
```

Comment:
Remove this line, because the default value of `d` is `null`.
The same as line 759.